docs/administration/audit-logs.md

```diff
@@ -26,16 +26,30 @@ Audit logs record the following changes:
     - Experiment created
     - Experiment configuration change 
     - Experiment metric changes
-    - Experiment manual update
+    - Experiment refresh changes
 - Metrics
-    - Name change
-    - Fact change
-    - Aggregation change
-    - Additional details (metric properties, winsorization, etc)
-- Facts
+    - Metric created
+    - Metric updated
+    - Metric collection changes
+- Definitions
     - FactSQL changes
-    - Fact create, edit, delete
+    - AssignmentSQL changes
+    - PropertySQL FactSQL changes
+    - Entity changes
 - Feature Flags
     - Creation
     - Update
-    - Deletion
\ No newline at end of file
+    - Deletion
+Allocation,  
+  Experiment,  
+  ExperimentMetricGroup,  
+  ExperimentMetricGroupMetric,  
+  ExperimentVariation,  
+  ExposureSchedule,  
+  FunnelMetricAggregation,  
+  Metric,  
+  MetricCollectionMetric,  
+  MetricEventAggregation,  
+  MetricEventEntity,  
+  MetricEventMeasure,  
+  MetricEventSource,  
```

Comment:
formatting here is off, i think you want to prepend dashes as above and remove the commas